packages/core/src/app/payment/paymentMethod/PaymentMethod.tsx

```diff
@@ -127,7 +127,8 @@ const PaymentMethodComponent: FunctionComponent<
         method.id === PaymentMethodId.OrbitalGooglePay ||
         method.id === PaymentMethodId.StripeGooglePay ||
         method.id === PaymentMethodId.StripeUPEGooglePay ||
-        method.id === PaymentMethodId.WorldpayAccessGooglePay
+        method.id === PaymentMethodId.WorldpayAccessGooglePay ||
```

Comment:
We should not be adding code here, instead better to start a separate package.

Reply:
I'll refactor GooglePay component to separate module in coming days.